l5kit/l5kit/data/zarr_utils.py

```diff
@@ -222,3 +223,61 @@ def zarr_split(input_zarr: str, output_path: str, split_infos: List[dict]) -> Li
         cur_scene = end_cut
 
     return cuts_track
+
+
+def zarr_scenes_chunk(input_zarr: str, output_zarr: str, num_frames_to_copy: int) -> None:
+    """
+    Copy `num_frames_to_keep` from each scene in input_zarr and past them into output_zarr
+
+    Args:
+        input_zarr (str): path to the input zarr
+        output_zarr (str): path to the output zarr
+        num_frames_to_copy (int): how many frames to copy from the start of each scene
+
+    Returns:
+
+    """
+    input_dataset = ChunkedDataset(input_zarr)
+    input_dataset.open()
+
+    # check we can actually copy the frames we want from each scene
+    assert np.all(np.diff(input_dataset.scenes["frame_index_interval"], 1) > num_frames_to_copy), "not enough frames"
```

Comment:
I know I'm annoying you a lot with things like this xD but shouldn't it be >= ?

```
indices = np.asarray([1, 5, 10])
np.diff(indices, 1)
```

gives [4, 5], as one bound is inclusive -> we can copy 4 frames from each scene?

Reply:
yes I know, it was more a question of:
- why would you use this to copy the entire zarr xD?

Reply:
Ah true, that's when it happens xD Minor, but still, I think it would be unexpected behavior with > so I would change it.